### PR TITLE
[WIP]Add testcases for libssh transport

### PR DESCRIPTION
**Pull request by ganeshrn**
Depends-On: https://github.com/ansible-collections/ansible.netcommon/pull/30

##### SUMMARY

*  Add test cases with ansible_network_cli_ssh_type set to libssh
   to test libssh connection plugin

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ganeshrn** [comment]
rechec

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

File: tests/integration/targets/eos_acl_interfaces/tasks/cli.yaml [hunk 1]
```diff
@@ -27,3 +27,12 @@
   with_items: '{{ test_items }}'
```
Comment (Qalthos) [inline tests/integration/targets/eos_acl_interfaces/tasks/cli.yaml:27]: It's not necessary, but we could be clearer about what we intend to test here

```suggestion
    ansible_network_cli_ssh_type: paramiko
  with_items: '{{ test_items }}'
```

**Comment by gundalow** [comment]
I see the dependency https://github.com/ansible-collections/ansible.netcommon/pull/30 got merged.
Is this still needed, otherwise could it please be closed. Thanks!

**Comment by ganeshrn** [comment]
Closing the PR as adding libssh test case will be handled by adding a zuul job

**Pull request closed without merging**